src/utils/metrics.ts

```diff
@@ -1,94 +1,151 @@
+});
+
+const merkleRootTotal = new client.Counter({
+  name: "watch_tower_merkle_roots_total",
+  help: "Total number of merkle roots processed",
+  labelNames: ["chain_id"],
+});
+
+const addContractsErrorsTotal = new client.Counter({
+  name: "watch_tower_add_contracts_errors_total",
+  help: "Total number of add contracts errors",
+  labelNames: ["chain_id", "function"],
+});
+
+const addContractsRunDurationSeconds = new client.Histogram({
+  name: "watch_tower_add_contracts_run_duration_seconds",
+  help: "Duration of add contracts run",
+  labelNames: ["chain_id"],
+});
+
+const processBlockDurationSeconds = new client.Histogram({
+  name: "watch_tower_process_block_duration_seconds",
+  help: "Duration of process block",
+  labelNames: ["chain_id"],
+});
+
+const totalActiveOwners = new client.Gauge({
+  name: "watch_tower_active_owners_total",
+  help: "Total number of owners with active orders",
+  labelNames: ["chain_id"],
+});
+
+const totalActiveOrders = new client.Gauge({
+  name: "watch_tower_active_orders_total",
+  help: "Total number of active orders",
+  labelNames: ["chain_id"],
+});
+
+const totalOrderBookDiscreteOrders = new client.Counter({
+  name: "watch_tower_orderbook_discrete_orders_total",
+  help: "Total number of discrete orders posted to the orderbook",
+  labelNames: ["chain_id", "handler", "owner", "id"],
+});
+
+const totalOrderBookErrors = new client.Counter({
+  name: "watch_tower_orderbook_errors_total",
+  help: "Total number of errors when interacting with the orderbook",
+  labelNames: ["chain_id", "handler", "owner", "id", "status", "error"],
+});
+
+const totalPollingRuns = new client.Counter({
+  name: "watch_tower_polling_runs_total",
+  help: "Total number of polling runs",
+  labelNames: ["chain_id", "handler", "owner", "id"],
+});
+
+const totalPollingOnChainChecks = new client.Counter({
+  name: "watch_tower_polling_onchain_checks_total",
+  help: "Total number of on-chain hint checks",
+  labelNames: ["chain_id", "handler", "owner", "id"],
+});
+
+const pollingOnChainDurationSeconds = new client.Histogram({
+  name: "watch_tower_polling_onchain_duration_seconds",
+  help: "Duration of on-chain hint checks",
+  labelNames: ["chain_id", "handler", "owner", "id"],
+});
+
+const totalPollingOnChainInvalidInterfaces = new client.Counter({
+  name: "watch_tower_polling_onchain_invalid_interface_total",
+  help: "Total number of invalid on-chain hint interface",
+  labelNames: ["chain_id", "handler", "owner", "id"],
+});
+
+const totalPollingOnChainEthersErrors = new client.Counter({
+  name: "watch_tower_polling_onchain_ethers_errors_total",
+  help: "Total number of ethers on-chain hint errors",
+  labelNames: ["chain_id", "handler", "owner", "id"],
+});
+
+const totalPollingUnexpectedErrors = new client.Counter({
+  name: "watch_tower_polling_unexpected_errors_total",
+  help: "Total number of unexpected polling errors",
+  labelNames: ["chain_id", "handler", "owner", "id"],
+});
+
+export {
```

Comment:
why export all the metrics like this? i think is best you add `export` word left of each `const` declaration above. 

The effect is the same, but you can benefit from tree-shaking for example if you just need a few metrics

Reply:
Fixed.